dynawo/sources/Common/Dictionaries/DYNError_en_GB.dic

```diff
@@ -26,6 +26,7 @@ OpenFileFailed              =             failed to open file %1%
 NoJobDefined                =             no job found in the .jobs file
 IncorrectDelay              =             inconsistent delay %1% at time %2% (max delay is %3%)
 IterationStepAndTimeStepBothDefined =     iteration step and time step can't be defined at the same time
+UnknownCriteriaType         =             chosen criteria type is unknown
```

Comment:
the XML parsing is doing that normally, no need to add a new error

Reply:
I mean that it is supposed to be tested during the parsing of the crt file, it does not belong to DYNCriteria.cpp to test inputs validity

Reply:
Maybe keeping the error is a bit more safe I don't know... I can withdraw the error if you want.

Reply:
Yes please remove it as it does not belong there